test/rack_utils.go

```diff
@@ -472,6 +472,13 @@ func getClusterNamespacedName(name, namespace string) types.NamespacedName {
 	}
 }
 
+func getPVCNamespacedName(name, namespace string) types.NamespacedName {
```

Comment:
Can create a generic func called `getNamespacedName` which will return `types.NamespacedName` for any object

Reply:
Can rename the existing func`getClusterNamespacedName` to `getNamespacedName`